src/test/java/org/springframework/integration/aws/inbound/KinesisMessageDrivenChannelAdapterTests.java

```diff
@@ -32,7 +32,6 @@
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-
```

Comment:
This is not OK from our Checkstyle requirements.
Please, consider to reinstate the empty like and call `gradlew clean check` locally.
I hope you have a Docker to be sure in the proper tests execution.